### DISCUSSION TITLE
How to connect Zapier with budibase

### DISCUSSION THREAD

**Discussion by AryanGaur14** [Budibase/budibase#13325] [Help] [answered]
I want to learn how to connect Zapier with Budibase because Budibase is not available on the Zapier platform.

**Comment by andz-bb** [comment, marked as answer]
hey @AryanGaur14 - you will need to use Webhooks on the Zapier side. [Here's a full guide on how to do it from our docs.](https://docs.budibase.com/docs/zapier) Hope this helps!
